.github/CONTRIBUTING.rst

```diff
@@ -2,26 +2,159 @@
 Contributing to pydov
 =====================
 
-Contribution is not only code implementation!
----------------------------------------------
+First of all, thanks for considering contributing to pydov! It's people like you that make it
+rewarding for us - the project :ref:`authors` - to work on pydov.
```

Comment:
- the project :ref: 'authors' - does not show nicely in html

Reply:
Will be ok in the rendered version of sphinx.